web/src/components/contact/ContactDialog.vue

```diff
@@ -23,20 +23,35 @@
         We'll send a message to contact {{ speaker.name }} on your behalf.
 
         <v-text-field
-          ref="name"
           v-model="form.name"
           label="Your Name"
+          :error-messages="nameErrors"
+          @input="$v.form.name.$touch()"
+          @blur="$v.form.name.$touch()"
```

Comment:
Just curious cause I never used Vue Validations, why do we need touch on all fields?

Reply:
I'm following the instructions from Vuelidate: https://vuelidate.js.org/#sub-without-v-model

There's some subtlety to getting it to validate when the user has left the field rather than aggressively validating as soon as input happens. Still polishing a bit.